src/main/scala/is/hail/annotations/MemoryBlock.scala

```diff
@@ -76,6 +88,11 @@ final class MemoryBlock(val mem: Array[Long]) {
     Platform.copyMemory(bytes, Platform.BYTE_ARRAY_OFFSET, mem, Platform.LONG_ARRAY_OFFSET + off, bytes.length)
   }
 
+  def storeBytes(off: Int, bytes: Array[Byte], size: Int) {
```

Comment:
I think this is actually "consistent".  size modifies bytes (the source) here.  In `loadBytes(off, size, bytes)`, n modifies `this` (the source), therefore it goes with off, before the destination.